js/base/Exchange.js

```diff
@@ -1206,8 +1206,8 @@ module.exports = class Exchange {
         return this.extend (order, {
             'id': this.safeString (order, 'id'),
             'clientOrderId': this.safeString (order, 'clientOrderId'),
-            'timestamp': datetime,
-            'datetime': timestamp,
+            'timestamp': timestamp,
```

Comment:
Hi @escoman

The `safeOrder` function will be compiled from js to python/php. Would you like to keep js updates?

BTW, you can update local php library with `npm run build` (if node was installed).

Reply:
@sc0Vu, hello.

So, for the correct changes, I have to correct [js/base/exchange.js], and then start `npm run build`, which will make changes to Python/PHP?

Reply:
@escoman all good now, will be merged soon thanks!